deploy/kubernetes/releases/csi-digitalocean-dev/driver.yaml

```diff
@@ -38,6 +38,36 @@ deletionPolicy: Delete
 
 ---
 
+kind: StorageClass
```

Comment:
Given that we're already defining quite a few storageclasses, I _think_ it might be better to instead document this feature, rather than explicitly define it for all future clusters.